lib/data/species/coldcorals/citations-utf8.csv

```diff
@@ -0,0 +1,67 @@
+14,Global,Chapter 42: Cold-water corals,E. Cordes,S. Arnaud-Haond,et al,2016,United Nations,https://www.un.org/Depts/los/global_reporting/WOA_RPROC/Chapter_42.pdf
+15,Chile,Large assemblages of cold-water corals in Chile: a summary of recent findings and potential impacts,V. Haussermann,G. Forsterra,,2007,"George, R. Y. and S. D. Cairns, eds. 2007. Conservation and adaptive 195 management of seamount and deep-sea coral ecosystems. Rosenstiel School of Marine and Atmospheric Science, University of Miami.",https://www.ingentaconnect.com/contentone/umrsmas/bullmar/2007/00000081/a00103s1/art00018?crawler=true&casa_token=vepkYQiSQF0AAAAA:X7IqHUJ6fnC6TFNEk3GZaH9uV38XV5ZuFNmXeTjrUcChRPiuj7u9Xn5Lp9sukpJ33FKPME_Dq1AE0J0c
+16,,Memoirs of the Hourglass Cruises,S. D. Cairns,,,1977,Marine Research Laboratory Florida Department of Natural Resources,https://repository.si.edu/bitstream/handle/10088/7219/IZ_1977_Cairns_Hourglass.pdf
+17,Mesoamerican Reef,Exploration and Mapping of the deep Mesoamerican Reef,P. J. Etnoyer,M. L. Brennan,et al.,2015,Oceanography,https://www.researchgate.net/profile/Michael_Brennan7/publication/274072842_Exploration_and_Mapping_of_the_Deep_Mesoamerican_Reef/links/55144f050cf2eda0df30bc86.pdf
+18,Brazil,"Deep-water octocorals (Cnidaria: Octocorallia) from Brazil: Family Chrysogorgiidae Verrill, 1883",R. T. S. Cordeiro,C. B. Castro,C. D. Perez,2015,Zootaxa,https://www.researchgate.net/profile/Carlos_Perez67/publication/286920358_Deep-water_octocorals_Cnidaria_Octocorallia_from_Brazil_Family_Chrysogorgiidae_Verrill_1883/links/56701b0408aececfd5530dd6/Deep-water-octocorals-Cnidaria-Octocorallia-from-Brazil-Family-Chrysogorgiidae-Verrill-1883.pdf
+19,Italy,High-resolution trace and minor element compositions in deep-water scleratctinian corals (Desmophyllum dianthus) from the Mediterranean Sea and the Great Australian Bight,P. Montagna,M. McCulloch,et al.,2005,Cold-water corals and ecosystems,https://www.researchgate.net/profile/Marco_Taviani/publication/227048932_High-resolution_trace_and_minor_element_compositions_in_deep-water_scleractinian_corals_Desmophyllum_dianthus_from_the_Mediterranean_Sea_and_the_Great_Australian_Bight/links/0912f4ffacaef5bc0d000000.pdf
+20,North-East Atlantic - potentially a lot of data!!!,"Deep-sea coral distribution on seamounts, oceanic islands, and continental slopes in the Northeast Atlantic",J. Hall-Spencer,A. Rogers,et al.,2007,"Conservation and adaptive management of seamount and deep-sea coral ecosystems. Rosenstiel School of Marine and Atmospheric Science, University of Miami",https://www.ingentaconnect.com/content/umrsmas/bullmar/2007/00000081/a00103s1/art00013?crawler=true
+21,Pacific,"Forever in the dark: the cave-dwelling azooxanthellate reef coral Leptoseris troglodyta sp. N. (Scleractinia, Agariciidae)",B. W. Hoeksema,,,2012,Zookeys,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3487639/
+,,,,,,,,
+,,,,,,,,

```

Comment:
I have noticed a lot of blank records from line 23 onwards and wondered if that was meant to be like that?